### PR TITLE
Change ArgParse to Click

### PR DESCRIPTION
**Pull request by rhenter**
PR with a refactor changing native ArgParser lib to Click lib and fix the config file uses. Following click documentation:

http://click.pocoo.org/5/

 - Command using help option:
```shell
$ translate-cli --help
Usage: __main__.py [OPTIONS] TEXT...

  Python command line tool to make on line translations

  Example:

       $ translate-cli -t zh the book is on the table
       碗是在桌子上。

  Available languages:

       https://en.wikipedia.org/wiki/ISO_639-1
       Examples: (e.g. en, ja, ko, pt, zh, zh-TW, ...)

Options:
  --version        Show the version and exit.
  -f, --from TEXT  Sets the language of the text being translated. The default
                   value is 'autodetect'
  -t, --to TEXT    Sets the language you want to translate.
  --help           Show this message and exit.
```
 - Command using version option:
```shell
$ translate-cli  --version
translate, version 3.3.0
```

@terryyin I also create a sample from config file to be used in the project documentation

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,8 +1,21 @@
 The MIT License (MIT)
-Copyright © 2017 @terryyin
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright © 2017 @terryyin (Translate)
 
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+Permission is hereby granted, free of charge, to any person obtaining a copy of
```
Comment (rhenter) [inline LICENSE]: Refactor by PEP8 recomendation

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[aliases]
+test=pytest --addopts '-vv tests'
+
+[bdist_wheel]
+universal=1
```
Comment (rhenter) [inline setup.cfg]: this means woks to python 2.x and 3.x without any code changes

File: tests/test_main.py [hunk 3]
```diff
@@ -1,28 +1,25 @@
 #!/usr/bin/env python
 # encoding: utf-8
-import sys
+from __future__ import unicode_literals
```
Comment (rhenter) [inline tests/test_main.py]: this import is because a python 2.x unicode problem

File: translate/version.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__version__ = "3.3.0"
```
Comment (rhenter) [inline translate/version.py]: this is generate automatically when you create a dist package

**Comment by terryyin** [comment]
shall I do a new release now?

**Comment by rhenter** [comment]
@terryyin yes please !

**Pull request merged**